### ISSUE TITLE
turn off the choke damage display on the armor stand

### ISSUE THREAD

**Issue by HANNAAN1288** [MagicCheese1/Damage-Indicator#26]
I would like to ask a question, can I turn off the choke damage display on the armor stand?

**Comment by MagicCheese1** [comment]
I don't really know what you mean. Are you sure you're really using my plugin? (I get a lot of people who using a different damage indicator plugin but ask in my GitHub/Discord.)

**Comment by MagicCheese1** [comment]
Or could you send some screenshot or more information about your problem

**Comment by HANNAAN1288** [comment]
Like this.
The armor stand will display damage by suffocation.
![image](https://github.com/MagicCheese1/Damage-Indicator/assets/108947333/2d75c3a0-0137-4295-a68e-ae8df208b696)

**Comment by MagicCheese1** [comment]
Do you have showToDamagerOnly set to false?

**Comment by HANNAAN1288** [comment]
Yes. But I just want to close suffocation damage from armor stand, want let other mobs' suffocation damage can display.
![image](https://github.com/MagicCheese1/Damage-Indicator/assets/108947333/f8bd2450-940f-4b01-9fd7-3b66bbbd9c6e)
![image](https://github.com/MagicCheese1/Damage-Indicator/assets/108947333/00a48a27-af3a-4954-b666-61c3be6d6d39)

**Comment by MagicCheese1** [comment]
This is very weird. My plugin only shows indicators for damage dealt by players and I think armorstands can't suffocate(?).
Are you sure you're using my plugin? I get a lot of people reporting issues even though they have a different damage indicator plugin.

**Comment by HANNAAN1288** [comment]
Yeah. I know.
But I would like to display damage indicators by suffocation on mobs and players, except the armor stand.XD
I'm sorry if you misunderstood me.
